### ISSUE TITLE
Modified toolbars (popup) and other (user-defined) manipulations

### ISSUE THREAD

**Issue by rexagod** [publiclab/Leaflet.DistortableImage#170]
Follow up on [140](https://github.com/publiclab/Leaflet.DistortableImage/issues/140#issuecomment-473101722) and [155](https://github.com/publiclab/Leaflet.DistortableImage/pull/155#discussion_r262564253).

- Allow "popup" toolbar manipulations (#155) -- can be built on top of #155
- Allow `Toolbar` "to be turned off and/or replaced with another toolbar-like UI entirely" (#140) -- can be employed using the "reveal" API, also in #155
- Expose a semantic API for the same, that allows different `position`, CSS styles, etc (#155) -- should be rigid as well, build from scratch? This one needs some brainstorming to finalize future plans.
- I think we should consider building this on #158 as well to include Multiple Image support in the process, maybe that'll ease out things, or open up corner cases that nonetheless need to be considered, and we might as well do that now.
***

/cc @jywarren @sashadev-sky

**Comment by sashadev-sky** [comment]
@rexagod agreed! I have been waiting to start working on UI and functionalities for these types of things until multiple image select is working. It is now actually a working implementation if you want to fetch my PR and see how it works to help you plan. (For now is hard coded with two images but changing that hopefully today). The page is `http://127.0.0.1:60179/examples/select.html` instead of the default index page. And use command + click because box zoom is still using old implementation not working yet.
